cmd/snap/cmd_auto_import.go

```diff
@@ -153,6 +148,20 @@ func init() {
 	cmd.hidden = true
 }
 
+func isOwned() (bool, error) {
```

Comment:
I mostly added it to avoid an error from running "create-user --knonw" if the device is already owned. However I can also do this by checking (and setting) the right error kind in the reply from the daemon and its more correct this way. So I removed "isOwned" now.